scripts/pod_lib_lint.rb

```diff
@@ -48,7 +50,21 @@ def main(args)
 
   command.push(*args)
   puts command.join(' ')
-  exec(*command)
+
+  # Run the lib lint command in a thread.
+  t = Thread.new do
+    system(*command)
+  end
+
+  # Print every minute since linting can run for >10m without output.
+  number_of_times_checked = 0
+  while t.alive? do
+    sleep 1.0
```

Comment:
Do the interruptions impact performance?  Should it be `sleep 10.0` or `sleep 60.0`?

Reply:
Tests were no slower or faster in terms of minutes, so it's probably fine. `sleep 60.0` would probably not be fine.